gateway/otel.go

```diff
@@ -0,0 +1,28 @@
+package gateway
+
+import (
+	"github.com/sirupsen/logrus"
+
+	tyktrace "github.com/TykTechnologies/opentelemetry/trace"
+)
+
+func (gw *Gateway) InitOpenTelemetry() {
+	gwConfig := gw.GetConfig()
+
+	traceLogger := mainLog.WithFields(logrus.Fields{
+		"exporter":           gwConfig.OpenTelemetry.Exporter,
+		"endpoint":           gwConfig.OpenTelemetry.Endpoint,
+		"connection_timeout": gwConfig.OpenTelemetry.ConnectionTimeout,
```

Comment:
I think we dont really need the timeout printed in each log?

Reply:
yes, acceptance criteria of the ticket